### ISSUE TITLE
Unable to generate ACME certificate in KV store after migration to ACME V2

### ISSUE THREAD

**Issue by frenchviking** [containous/traefik#3372]
### Do you want to request a *feature* or report a *bug*?

a bug
### What did you do?

Upgraded from 1.5.3 to 1.6.1.

### What did you expect to see?

Using the onHost rule, I expect to get a certificate for my domain.

### What did you see instead?

```
May 23rd 2018, 11:53:35.000 | time="2018-05-23T09:53:35Z" level=error msg="Error adding ACME certificate for domain [\"*.mydomain.io\" \"mydomain.io\"]: tls: failed to find any PEM data in certificate input"
May 23rd 2018, 11:53:34.000 | time="2018-05-23T09:53:34Z" level=error msg="Datastore sync error: Object lock value: expected 26552e5d-acd1-4686-b2c9-f1af701f227d, got 457e9bd6-0a4a-437b-b9de-62a97cc5a3ef, retrying in 349.512489ms"
May 23rd 2018, 11:53:33.000 | legolog: 2018/05/23 09:53:33 [INFO][*.mydomain.io, mydomain.io] acme: Obtaining bundled SAN certificate
May 23rd 2018, 11:53:32.000 | time="2018-05-23T09:53:32Z" level=error msg="Datastore sync error: Object lock value: expected 457e9bd6-0a4a-437b-b9de-62a97cc5a3ef, got a74a7c07-8ab5-40cb-b995-1a9ff12da459, retrying in 601.579434ms"
```

Same issue with DNS-01 challenge and AWS provider, but the CNAME DNS validation record got created.

Rolling back to 1.5.3, got my certs.
### Output of `traefik version`: (_What version of Traefik are you using?_)

```
1.6.1
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

Docker Swarm backend
Configuration store :+1:  Consul KV
Arguments: 
```
 -consul --consul.endpoint=admin_consul:8500 --consul.prefix=traefik
```
Docker service environments : 
```
AWS_SECRET_ACCESS_KEY=xxx AWS_ACCESS_KEY_ID=xxx AWS_HOSTED_ZONE_ID=xxx
```

```
traefik/acme/:
traefik/acme/account/lock:a74a7c07-8ab5-40cb-b995-1a9ff12da459
traefik/acme/account/object:[BINARY CERT INFO]
traefik/acme/acmelogging:true
traefik/acme/caserver:
traefik/acme/dnschallenge/delayBeforeCheck:0
traefik/acme/dnschallenge/provider:route53
traefik/acme/domains/0/main:*.mydomain.io
traefik/acme/domains/0/sans/0:mydomain.io
traefik/acme/email:contact@mydomain.com
traefik/acme/entrypoint:https
traefik/acme/httpchallenge/:
traefik/acme/httpchallenge/entrypoint:http
traefik/acme/ondemand:false
traefik/acme/onhostrule:true 
traefik/acme/storage:traefik/acme/account

```

### If applicable, please paste the log output at DEBUG level (`--logLevel=DEBUG` switch)

```
(paste your output here)
```

**Comment by nmengin** [comment]
After a deep analyze of the problem, it seems to be problems during migration from V1.5 to 1.6.

- Certificates with null value are not deleted as with a acme file (cf #3191).
- Account seems to not be correctly reset.

**Comment by traefiker** [comment]
Closed by #3389.

**Comment by holms** [comment]
I still got this problem :( ``` level=error msg="Datastore sync error: Object lock value: expected 942ffd74-b4c6-4255-95ce-f483398314dc, got b1cb7559-1db0-4d06-9bfa-63968adf0cc6, retrying in 674.994029ms"
```
